lib/twilioconnection.js

```diff
@@ -435,7 +445,7 @@ class TwilioConnection extends StateMachine {
  * {@link TwilioConnection} options
  * @typedef {object} TwilioConnectionOptions
  * @property {LogLevel} [logLevel=warn] - Log level of the {@link TwilioConnection}
- * @property {number} [maxConsecutiveMissedHeartbeats=5] - Max. number of consecutive "heartbeat" messages that can be missed
+ * @property {number} [maxConsecutiveMissedHeartbeats=3] - Max. number of (effective) consecutive "heartbeat" messages that can be missed
```

Comment:
Is the 5 just a documentation a typo, while DEFAULT_MAX_CONSECUTIVE_MISSED_HEARTBEATS represents the correct value of 3?

Reply:
yes, seems like just documentation typo.